runt.toml

```diff
@@ -469,26 +456,24 @@ expect_dir = "tests/correctness/systolic/mmult-expect"
 name = "[correctness] `@fast` attribute should not change program behavior"
 paths = ["tests/correctness/fast-attr/valid/*.futil"]
 cmd = """
-fud exec --from calyx --to jq \
-         --through dat \
-         --through icarus-verilog \
-         -s calyx.flags "-p validate -p compile -p lower -d static-promotion" \
-         -s verilog.data {}.data \
-         -s jq.expr "." \
+fud2 --from calyx --to jq \
+         --through icarus \
+         -s sim.data={}.data \
+         -s calyx.args="-p validate -p compile -p lower -d static-promotion" \
+         -s jq.expr="." \
          {} -q
 """
 
 [[tests]]
 name = "[correctness] `@fast` attribute should reject unoptimizable sequences"
 paths = ["tests/correctness/fast-attr/invalid/*.futil"]
 cmd = """
-fud exec --from calyx --to jq \
-         --through dat \
-         --through icarus-verilog \
-         -s calyx.flags "-p validate -p compile -p lower -d static-promotion" \
-         -s verilog.data {}.data \
-         -s jq.expr "." \
-         {} -q
+RUST_LIB_BACKTRACE=0 fud2 --from calyx --to jq \
```

Comment:
I see why this is necessary, but it's certainly weird! As another minor to-do for some future PR, maybe we can just make this error print something more sensible that doesn't include a stacktrace in the first place.

Reply:
Yeah I'm not sure why it works differently on CI, probably a setting we can fix somewhere, but this works for the moment